components/sidebar/sidebar_channel/sidebar_channel.jsx

```diff
@@ -257,6 +257,8 @@ export default class SidebarChannel extends React.PureComponent {
                     offTopicDisplayName={this.props.offTopicDisplayName}
                 />
             );
+            // FIXME most likely cause for a warning:
```

Comment:
Good find. This is definitely incorrect. If this component is indeed responsible for this, it should be triggered in `componentDidMount` or `componentDidUpdate` based on some change in the other props

Reply:
If its ok I will leave the comment for now and come back to it later, as I need to focus on the refactor.

Reply:
Yep, that's totally fine. I'll file a ticket

Edit: https://mattermost.atlassian.net/browse/MM-21052